deployment/DB_VERSION

```diff
@@ -0,0 +1 @@
+69
```

Comment:
I shall make sure this is in sync with master before merge

Reply:
71 in master now.

Reply:
sync in 2ea24f18